include/bluetooth/audio/aics.h

```diff
@@ -0,0 +1,450 @@
+ *
+ * @return 0 on success, GATT error value on fail.
+ */
+int bt_aics_state_get(struct bt_conn *conn, struct bt_aics *inst);
+
+/**
+ * @brief Read the Audio Input Control Service gain settings.
+ *
+ * @param conn          Connection to peer device, or NULL to read local server value.
+ * @param inst          Pointer to the Audio Input Control Service instance.
+ *
+ * @return 0 on success, GATT error value on fail.
+ */
+int bt_aics_gain_setting_get(struct bt_conn *conn, struct bt_aics *inst);
+
+/**
+ * @brief Read the Audio Input Control Service input type.
+ *
+ * @param conn          Connection to peer device, or NULL to read local server value.
+ * @param inst          Pointer to the Audio Input Control Service instance.
+ *
+ * @return 0 on success, GATT error value on fail.
+ */
+int bt_aics_type_get(struct bt_conn *conn, struct bt_aics *inst);
+
+/**
+ * @brief Read the Audio Input Control Service input status.
+ *
+ * @param conn          Connection to peer device, or NULL to read local server value.
+ * @param inst          Pointer to the Audio Input Control Service instance.
+ *
+ * @return 0 on success, GATT error value on fail.
+ */
+int bt_aics_status_get(struct bt_conn *conn, struct bt_aics *inst);
+
+/**
+ * @brief Unmute the Audio Input Control Service input.
+ *
+ * @param conn          Connection to peer device, or NULL to read local server value.
+ * @param inst          Pointer to the Audio Input Control Service instance.
+ *
+ * @return 0 on success, GATT error value on fail.
+ */
+int bt_aics_unmute(struct bt_conn *conn, struct bt_aics *inst);
+
+/**
+ * @brief Mute the Audio Input Control Service input.
+ *
+ * @param conn          Connection to peer device, or NULL to read local server value.
+ * @param inst          Pointer to the Audio Input Control Service instance.
+ *
+ * @return 0 on success, GATT error value on fail.
+ */
+int bt_aics_mute(struct bt_conn *conn, struct bt_aics *inst);
+
+/**
+ * @brief Set input gain to manual.
+ *
+ * @param conn          Connection to peer device, or NULL to set local server value.
+ * @param inst          Pointer to the Audio Input Control Service instance.
+ *
+ * @return 0 on success, GATT error value on fail.
+ */
+int bt_aics_manual_gain_set(struct bt_conn *conn, struct bt_aics *inst);
+
+/**
+ * @brief Set the input gain to automatic.
+ *
+ * @param conn          Connection to peer device, or NULL to set local server value.
+ * @param inst          Pointer to the Audio Input Control Service instance.
+ *
+ * @return 0 on success, GATT error value on fail.
+ */
+int bt_aics_automatic_gain_set(struct bt_conn *conn, struct bt_aics *inst);
+
+/**
+ * @brief Set the input gain.
+ *
+ * @param conn          Connection to peer device, or NULL to set local server value.
+ * @param inst          Pointer to the Audio Input Control Service instance.
+ * @param gain          The gain in dB to set (-128 to 127).
```

Comment:
This is not in dB, but in gain setting units, right?

Reply:
Uh, that's a very good point.